src/Raft.sol

```diff
@@ -0,0 +1,38 @@
+// contracts/NFT.sol
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+
+contract Raft is ERC721URIStorage, Ownable, Pausable {
```

Comment:
I think it's time we start documenting full-fledged comments using natspec 
https://docs.soliditylang.org/en/v0.8.15/natspec-format.html

Reply:
Let's add inline comments too to document why we're transferring ownership in ctor or setting "pause" for example

Reply:
done